src/CxbxKrnl/EmuD3D8.cpp

```diff
@@ -5503,110 +5573,39 @@ VOID WINAPI XTL::EMUPATCH(D3DDevice_EnableOverlay)
 // ******************************************************************
 VOID WINAPI XTL::EMUPATCH(D3DDevice_UpdateOverlay)
 (
-    X_D3DSurface *pSurface,
-    CONST RECT   *SrcRect,
-    CONST RECT   *DstRect,
-    BOOL          EnableColorKey,
-    D3DCOLOR      ColorKey
-)
+	X_D3DSurface *pSurface,
```

Comment:
Sorry about these unintended indenting-changes, I overlooked them when submitting (out of enthusiasm this PR actually works!)